Shared/gta/VehicleUpgrades.h

```diff
@@ -0,0 +1,223 @@
+/*****************************************************************************
```

Comment:
The file should have the extension .hpp to make it clear that it contains definitions and cannot be used as a "normal" header.

Reply:
Not always, but I agree with you. Done.